src/hotspot/share/libadt/dict.cpp

```diff
@@ -75,7 +75,21 @@ Dict::Dict(CmpKey initcmp, Hash inithash, Arena *arena, int size)
 
   _cnt = 0;                     // Dictionary is empty
   _bin = (bucket*)_arena->Amalloc_4(sizeof(bucket)*_size);
-  memset((void*)_bin,0,sizeof(bucket)*_size);
+  memset((void*)_bin, 0, sizeof(bucket) * _size);
+}
+
+// Deep copy into arena of choice
+Dict::Dict(const Dict &d, Arena *arena)
```

Comment:
Some minor formatting errors in this method: 
Arena * -> Arena*
void * -> void*
_cnt*2*sizeof -> _cnt * 2 * sizeof

Otherwise looks good!